lib/export_file.js

```diff
@@ -77,3 +77,12 @@ export async function exportMembers(collectiveSlug, tierSlug, options = { type:
   const date = formatDate(new Date());
   return exportFile('text/plain;charset=utf-8', `${date.replace(/-/g, '')}${path.replace(/\//g, '-')}`, csv);
 }
+
+export async function exportHosts(collectiveSlug, format = 'json') {
+  let path = `/v2/${collectiveSlug}/hosted`;
```

Comment:
If I test locally, this is a 404. This should be fetched using `REST_URL` (to be introduced, similar to IMAGES_URL or API_URL, default to `https://rest-staging.opencollective.com`).

Reply:
This uses the `get` function on api library. 
For the latest update I set `allowExternal` = `REST_URL`. This enabled fetching via the external URL, rather than, using `API Proxy`.